submissions/bmukha/hooli-style-popup/index.html

```diff
@@ -0,0 +1,205 @@
+                    /></a>
+                  </li>
+                  <li class="header__nav__item__main">
+                    <a href="#"
+                      ><img
+                        src="./popup-icons/files.png"
+                        width="48"
+                        alt="files"
+                    /></a>
+                  </li>
+                  <li class="header__nav__item__main">
+                    <a href="#"
+                      ><img
+                        src="./popup-icons/calendar.png"
+                        width="48"
+                        alt="calendar"
+                    /></a>
+                  </li>
+                  <li class="header__nav__item__main">
+                    <a href="#"
+                      ><img
+                        src="./popup-icons/documents.png"
+                        width="48"
+                        alt="documents"
+                    /></a>
+                  </li>
+                  <li class="header__nav__item__main">
+                    <a href="#"
+                      ><img src="./popup-icons/maps.png" width="48" alt="maps"
+                    /></a>
+                  </li>
+                  <li class="header__nav__item__main">
+                    <a href="#"
+                      ><img src="./popup-icons/chat.png" width="48" alt="chat"
+                    /></a>
+                  </li>
+                  <li class="header__nav__item__main">
+                    <a href="#"
+                      ><img src="./popup-icons/news.png" width="48" alt="news"
+                    /></a>
+                  </li>
+                  <li class="header__nav__item__main">
+                    <a href="#"
+                      ><img
+                        src="./popup-icons/contacts.png"
+                        width="48"
+                        alt="contacts"
+                    /></a>
+                  </li>
+                  <li class="header__nav__item__main">
+                    <a href="#"
+                      ><img
+                        src="./popup-icons/classroom.png"
+                        width="48"
+                        alt="classroom"
+                    /></a>
+                  </li>
+                </ul>
+                <input
+                  type="checkbox"
+                  name="more_checkbox"
+                  class="more__checkbox"
+                  id="more__checkbox"
+                />
+                <label
+                  for="more__checkbox"
+                  class="more__label"
+                  tabindex="5"
+                ></label>
+                <nav class="menu__wrapper__secondary">
+                  <ul class="menu__items__secondary">
+                    <li class="header__nav__item__secondary">
+                      <a href="#"
+                        ><img
+                          src="./popup-icons/apps.png"
+                          width="48"
+                          alt="apps"
```

Comment:
For icons it's better set an empty alt attribute, read more [here](https://accessibility.psu.edu/images/imageshtml/#empty)